### PR TITLE
HFEYP-171 hyphens

### PR DESCRIPTION
**Pull request by guy-roberts**
### Context
A fix for HFEYP-171, hyphens should be allowed in page titles, and preserved in slugs

### Changes proposed in this pull request
HFEYP-171
Also, RedCarpet is removed from Gemfile
Fixes a bug in the GovSpeak controller, where the markdown is blank (no Jira issue)
Simplify landing page ordering
Breadcrumb links work

### Guidance to review
Add a page with a hyphen in the title, save it
View the site and check that the page has the hyphen in the title, and that the slug includes it too
Check that the landing page order (left to right then down) is Comm, Physical, Personal, Literacy, maths, Understanding, Expressive
Check that the breadcrumb links navigate correctly

### CODE REVIEW

File: app/models/content_page.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class ContentPage < ApplicationRecord
   scope :top_level, -> { where("parent_id IS NULL") }
   scope :order_by_position, -> { order("position ASC") }
 
-  ONLY_ALPHA_NUMERIC_AND_SPACE = /\A[a-zA-Z0-9,. ]+\Z/.freeze
-  validates :title, format: { with: ONLY_ALPHA_NUMERIC_AND_SPACE, message: "should only contain alphabetic, numeric, comma, fullstop and space characters" }
+  ONLY_ALPHA_NUMERIC_AND_SPACE = /\A[a-zA-Z0-9,.\- ]+\Z/.freeze
```
Comment (tonyto) [inline app/models/content_page.rb:7]: Constant needs updating to reflect the change

File: spec/models/content_page_spec.rb [hunk 2]
```diff
@@ -30,9 +30,13 @@
     expect { page.save! }.to_not raise_error
   end
 
-  it "sets the slug from the title, removing fullstops" do
```
Comment (tonyto) [inline spec/models/content_page_spec.rb:33]: The full stop test still feels relevant and should be kept. What do you think?

**Review by tonyto — APPROVED** [review]
Couple of comments but otherwise 👍

File: spec/models/content_page_spec.rb (hunk 2 above)
Comment (guy-roberts) [reply spec/models/content_page_spec.rb:33]: Done

**Review by tonyto — APPROVED** [review]

**Pull request merged**